knowledge-base/customize-selected-tab-border-tabstrip-blazor.md

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
The "tab" word is ambiguous. It may mean the "button" at the top, or the "content area" at the bottom. This article seems to use "tab" for both at the same time, if I understand correctly. You may want to consider some clarifications.